lib/store/README.md

```diff
@@ -0,0 +1,122 @@
+
+Some notable parameters:
+
+- `parameters.fileName` - the file that the story was defined in, when available
+
+## Args
+
+Args are "inputs" to stories.
+
+You can think of them equivalently to React props, Angular inputs and outputs, etc.
+
+Changing the args cause the story to be re-rendered with the new set of args.
+
+### Using args in a story
+
+By default (starting in 6.0) the args will be passed to the story as first argument and the context as second:
+
+```js
+const YourStory = ({ x, y } /*, context*/) => /* render your story using `x` and `y` */
+```
+
+If you set the `parameters.options.passArgsFirst` option on a story to false, args are passed to a story in the context, preserving the pre-6.0 story API; like parameters, they are available as `context.args`.
+
+```js
+const YourStory = ({ args: { x, y }}) => /* render your story using `x` and `y` */
+```
+
+### Arg types and values
+
+Arg types are used by the docs addon to populate the props table and are documented there. They are controlled by `argTypes` and can (sometimes) be automatically inferred from type information about the story or the component rendered by the story.
+
+A story can set initial values of its args with the `args` annotation. If you set an initial value for an arg that doesn't have a type a simple type will be inferred from the value.
+
+If an arg doesn't have an initial value, it will start unset, although it can still be set later via user interaction.
+
+Args can be set at the project, component and story level.
+
+### Using args in an addon
+
+Args values are automatically synchronized (via the `changeStoryArgs` and `storyArgsChanged` events) between the preview and manager; APIs exist in `lib/api` to read and set args in the manager.
+
+Args need to be serializable -- so currently cannot include callbacks (this may change in a future version).
+
+Note that arg values are passed directly to a story -- you should only store the actual value that the story needs to render in the arg. If you need more complex information supporting that, use parameters or addon state.
+
+Both `@storybook/client-api` (preview) and `@storybook/api` (manager) export a `useArgs()` hook that you can use to access args in decorators or addon panels. The API is as follows:
+
+```js
+import { useArgs } from '@storybook/client-api'; // or '@storybook/api'
+
+// `args` is the args of the currently rendered story
+// `updateArgs` will update its args. You can pass a subset of the args; other args will not be changed.
+const [args, updateArgs] = useArgs();
+```
+
+## ArgTypes
+
+Arg types add type information and metadata about args that are used to control the docs and controls addons.
+
+### ArgTypes enhancement
+
+To add a argTypes enhancer, `export addArgTypesEnhancers = []` from `preview.js` or and addon
```

Comment:
Do we really want to document this?

Reply:
```suggestion
To add a argTypes enhancer, `export argTypesEnhancers = []` from `preview.js` or and addon
```

Reply:
Maybe not? I copied it over from `client-api` and updated it. WDYT?